wallet-new/test/unit/Test/Infrastructure/Genesis.hs

```diff
@@ -30,10 +33,13 @@ data GenesisValues h = GenesisValues {
 
       -- | Hash of the bootstrap transaction
     , hashBoot :: h (Transaction h Addr)
+
+      -- | Fee policy
+    , txFee :: Int -> [Value] -> Value
```

Comment:
Adding the fee policy here helps avoid many other `HasGenesisBlockVersionData` constraints.